### ISSUE TITLE
Post image - rss

### ISSUE THREAD

**Issue by Pixelalb** [krivaten/ghostTown#1]
Thank you for the code!
I know, ghost doesn't send post image related info to the rss feed. Would be nice to get the post image in that loop too though.

**Comment by krivaten** [comment]
I totally agree! I believe I read somewhere that they are working on opening up the API so that we would be able to pull all the post's information, but until then, we will have to make do with what is supplied via the RSS feed.

Depending on the size of your blog, requesting the RSS feed and filtering through it to get the content you want could be a very slow process. Will try looking for other solutions because I would love to have this feature as well!

**Comment by naumanahmed19** [comment]
any update on this?
